index.js

```diff
@@ -524,42 +522,37 @@ Lexer.prototype = {
    */
 
   include: function() {
-    var tok = this.scanEndOfLine(/^include +([^\n]+)/, 'include');
+    var tok = this.scan(/^include/, 'include');
     if (tok) {
       this.tokens.push(tok);
+      this.inInclude = true;
+      while (this.inInclude) {
+        this.filter() ||
+          this.attrs() ||
+          this.path() ||
+          this.fail();
+      }
```

Comment:
I wonder if this would be better off written as:

```js
while (this.filter({inInclude: true})) {
  // optional attrs
  this.attrs({inInclude: true});
}
this.path();
```

This has the advantage that `inInclude` no longer needs to be a global thing, so it localises this added complexity a bit better.  Another plus is that it only allows `attrs` immediately after `filter`, and it's clearer from reading this code that the `path` terminates this sequence.

Another thought, should we also be emitting two tokens for extends for consistency?